docs/chapters/sdk/installing-the-sdk.rst

```diff
@@ -0,0 +1,29 @@
+Downloading and installing the SDK
+==================================
+
+Download the SDK for your hardware from the `PELUX website`_, it is quite big because it contains tho whole sysroot for the project in addition to some native tools, so depending on your internet connectivity it could take a couple of minutes.
+
+If you download it from a HTTP server then the file per default is not executable, we need to change it with `chmod`. After that is done we can run it, it's a self extracting shell script.
```

Comment:
It seems all the rest of the SWF is not written like this but it also doesn't use `you` as often as I did. I will rewrite it and tone it down.